src/Tribe/Attendee_Repository.php

```diff
@@ -454,31 +457,45 @@ public function filter_by_order_status( $order_status ) {
 		/** @var wpdb $wpdb */
 		global $wpdb;
 
-		$statuses_in = "'" . implode( "','", array_map( [ $wpdb, '_escape' ], $statuses ) ) . "'";
+		$value_operator = 'IN';
+		$value_clause   = "( '" . implode( "','", array_map( [ $wpdb, '_escape' ], $statuses ) ) . "' )";
+
+		if ( 'not_in' === $type ) {
+			$value_operator = 'NOT IN';
+		}
 
 		$has_plus_providers = class_exists( 'Tribe__Tickets_Plus__Commerce__EDD__Main' )
 		                      || class_exists( 'Tribe__Tickets_Plus__Commerce__WooCommerce__Main' );
 
-		$this->filter_query->join( "LEFT JOIN {$wpdb->postmeta} order_status_meta "
-		                           . "ON {$wpdb->posts}.ID = order_status_meta.post_id" );
+		$this->filter_query->join( "
+			LEFT JOIN {$wpdb->postmeta} order_status_meta
+			ON order_status_meta.post_id = {$wpdb->posts}.ID
+		", 'order-status-meta' );
+
+		$et_where_clause = "
+			(
+				order_status_meta.meta_key IN ( '_tribe_rsvp_status', '_tribe_tpp_status' )
+				AND order_status_meta.meta_value {$value_operator} {$value_clause}
+			)
+        ";
```

Comment:
Spacing looks weird here? Should line up with `$et_where_clause` - no?

Reply:
I'm using extra spacing here for easier SQL debugging since these queries get hard to read as they get complicated

Reply:
I think he means the `";` bit.

Reply:
Oh I was following what wp phpcs was dictating in phpstorm on this, if that's wrong I can adjust the ruleset as necessary.

Reply:
Fixed, it was spaces and not tabs for some reason